Packs/MISP/.secrets-ignore

```diff
@@ -1,3 +1,9 @@
 5.6.4.4
 1.2.4.4
-1.5.52.1
\ No newline at end of file
+1.5.52.1
+209.85.128.198
+04.10.05.43
+CAPz_z5hBQLj98G50S_ST8UDRiW8b0Umtd1K-iVWPDTFKbzhN4g@mail.gmail.com
+2002:a25:aba6:0:b0:641:3b6f:41ac
+2002:a81:2592:0:b0:2d1:1210:275e
+2002:a5e:c913:0:0:0:0:0
```

Comment:
where did these ips come from? Is it dummy data?

Reply:
yes, from the "test email object.eml" file